core/src/main/java/com/microsoft/applicationinsights/extensibility/initializer/DeviceInfoContextInitializer.java

```diff
@@ -40,5 +40,11 @@ public void initialize(TelemetryContext context)
         device.setOperatingSystemVersion(DeviceInfo.getOperatingSystemVersion());
         device.setId(DeviceInfo.getHostName());
         device.setLocale(DeviceInfo.getLocale());
+
+        // remove this when setRoleInstance is removed from DeviceContext
+        String hostName = CommonUtils.getHostName();
```

Comment:
My specific point was for this revert.

Reply:
Right. Since they're default initializers, I'll remove this.